framework/src/loops/ComputeElemAuxBcsThread.C

```diff
@@ -79,8 +80,20 @@ ComputeElemAuxBcsThread::operator()(const ConstBndElemRange & range)
             needed_mat_props.insert(mp_deps.begin(), mp_deps.end());
           }
           _problem.setActiveMaterialProperties(needed_mat_props, _tid);
+
+          const Elem * neighbor = elem->neighbor_ptr(side);
+
           _problem.reinitMaterialsFace(elem->subdomain_id(), _tid);
+          if (neighbor != nullptr)
```

Comment:
Ideally we need to reinitialize material on the neighbour side to correctly get update value for  an interface material using getNeighborMatarialProperty(). This is the reason for this change, however it doesn't work